feature-libs/cart/tsconfig.lib.json

```diff
@@ -0,0 +1,30 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/lib",
+    "target": "es2015",
+    "module": "es2020",
+    "moduleResolution": "node",
+    "declaration": true,
+    "sourceMap": true,
+    "strict": false, // TODO: Fix strict errors and remove this option
```

Comment:
New libraries should use `"strict": true`

Reply:
changes to this will be found in https://github.com/SAP/spartacus/issues/11617